integration_tests/elemental_01.py

```diff
@@ -31,6 +31,63 @@ def verify2d(array: f64[:, :], result: f64[:, :], size1: i32, size2: i32):
         for j in range(size2):
             assert abs(cos(array[i, j])**2 - result[i, j]) <= eps
 
+
+def verify1d_sum(array_a: f64[:], array_b: f64[:], result: f64[:], size: i32):
+    i: i32
+    eps: f64
+    eps = 5e-6
+
+    for i in range(size):
+        assert abs(array_a[i]**2 + 5*array_b[i]**3 - result[i]) <= eps
+
+
+def verify1d_mul(array_a: f64[:], array_b: f64[:], result: f64[:], size: i32):
+    i: i32
+    eps: f64
+    eps = 5e-6
```

Comment:
I think this should be, `1e-12`. For `f64` the results are pretty accurate. Are they not in this case?